custom/panel_templates/Default/footer.tpl

```diff
@@ -1,11 +1,21 @@
 <footer class="sticky-footer bg-white">
-    <div class="container my-auto">
-        <div class="copyright text-center my-auto">
-            <span>{$PAGE_LOAD_TIME}</span>&nbsp;&nbsp;&nbsp;&nbsp;
-            <span>&copy; NamelessMC {'Y'|date}</span><br /><br />
-            {* We ask that you keep a link back to our website or Github somewhere on your website *}
-            <a href="https://github.com/NamelessMC/Nameless" target="_blank"><i class="fab fa-github fa-fw"></i> {$SOURCE}</a>&nbsp;&nbsp;&nbsp;&nbsp;
-            <a href="https://namelessmc.com" target="_blank"><i class="fas fa-life-ring fa-fw"></i> {$SUPPORT}</a>
+    <div class="footer">
+        <div class="d-flex">
+            <div class="col-sm-5 mb-0">
+                <span>{'Y'|date} &copy; <a href="https://namelessmc.com/" target="_blank">NamelessMC</a></span>
+            </div>
+            <div class="col-sm-2 text-center mb-0">
+                <span class="font-size-sa">{$PAGE_LOAD_TIME}</span>
+            </div>
+            <div class="col-sm-5 mb-0">
+                <div class="float-right">
+                    <a class="ml-1" href="https://github.com/NamelessMC/Nameless" target="_blank" data-toggle="tooltip" data-placement="top" title="{$SOURCE}"><i class="fab fa-github fa-fw text-github"></i></a>
+                    <a class="ml-1" href="https://discord.com/invite/3wzUeGjX?" target="_blank" data-toggle="tooltip" data-placement="top" title="Discord"><i class="fab fa-discord fa-fw text-discord"></i></a>
```

Comment:
or discird.gg/nameless :)

Reply:
Oh, didn't know there is such a link for namelessMC.